__tests__/app.tests.js

```diff
@@ -45,6 +45,49 @@ describe("GET /api", () => {
         .then((response) => {
             const body = response.body;
             expect(body).toEqual(endpoints);
-        })
-    })
-})
\ No newline at end of file
+        });
+    });
+});
+
+describe("GET /api/articles/:article_id", () => {
+    test("returns status code 200 & requested article", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+            const articles = response.body;
+            expect(articles.article_id).toBe(1);
+            expect(articles).toMatchObject({
+                article_id: expect.any(Number),
```

Comment:
You're not expecting any number or any string. You're specifically expecting the data that is represented by article 1. At the very least you know you should get the article_id of 1 in this case. If I wanted article 1 but was served article 13 in this case this test would pass and it shouldn't